### PR TITLE
Legacy import tasks

### PR DESCRIPTION
**Pull request by goose-life**

### CODE REVIEW

File: indigo_kenyalaw/importer.py [hunk 1]
```diff
@@ -152,11 +152,11 @@ def filter_to_point_in_time(self, act, pit):
             for elem, start, end in group:
                 if keep is None and pit >= start:
                     # target, keep this one
-                    log.info("Keeping  %s - %s" % (elem.get('id'), elem.get('period')))
+                    log.info("Keeping  %s – %s" % (elem.get('id'), elem.get('period')))
```
Comment (longhotsummer) [inline indigo_kenyalaw/importer.py]: At some point changing these may bite you/us - when we're searching through logs or other non UI elements, we're more likely to expect a hyphen than en/emdash. I'd suggest keeping these as hyphens.

File: indigo_kenyalaw/models.py [hunk 2]
```diff
@@ -114,9 +115,48 @@ def do_import(self, user):
...
+                self.create_review_task(doc, user)
 
         self.finished_at = datetime.datetime.now()
         self.save()
 
+    def create_review_task(self, doc, user):
+        try:
+            # This doc already has an open review task, don't create another
+            Task.objects.get(
+                title='Review automatically created expression',
```
Comment (longhotsummer) [inline indigo_kenyalaw/models.py]: You need this to match, I suggest a constant on the class.

```python
class Foo(...):
  TASK_TITLE = 'foo'

  def bar(self, ...):
    thing = self.TASK_TITLE
```

File: indigo_kenyalaw/importer.py (hunk 1 above)
Comment (goose-life) [reply indigo_kenyalaw/importer.py]: good point!

**Comment by goose-life** [comment]
closes #10

**Pull request merged**